ssl/ssl_x509.cc

```diff
@@ -160,13 +160,15 @@ BSSL_NAMESPACE_BEGIN
 // installed. Calling an X509-based method on an |ssl| with a different method
 // will likely misbehave and possibly crash or leak memory.
 static void check_ssl_x509_method(const SSL *ssl) {
-  assert(ssl == NULL || ssl->ctx->x509_method == &ssl_crypto_x509_method);
+  assert(ssl == NULL || (ssl != NULL && ssl->ctx != NULL &&
```

Comment:
Should we be using nullptr over NULL as we rework code? I notice further down we use nullptr for the check rather then NULL.

Reply:
I'm on team nullptr as well for C++. I got sucked into following the original format for this, but I'll make the change 👍